__tests__/app.test.js

```diff
@@ -83,4 +83,34 @@ describe("GET /api/articles/:article_id", () => {
             expect(response.body.msg).toBe('Bad request');
           });
       });
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles", () => {
+    test("responds with an array of article objects", () => {
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(Array.isArray(articles)).toBe(true);
+          articles.forEach((article) => {
+              expect(article).toHaveProperty('author');
+              expect(article).toHaveProperty('title'); 
+              expect(article).toHaveProperty('article_id');
+              expect(article).toHaveProperty('topic'); 
+              expect(article).toHaveProperty('created_at'); 
+              expect(article).toHaveProperty('votes');
+              expect(article).toHaveProperty('article_img_url');
+              expect(article).toHaveProperty('comment_count');          
```

Comment:
Using a different matcher like objectContaining would be better here.
https://jestjs.io/docs/expect